lbclient.go

```diff
@@ -98,6 +98,18 @@ func (cc *LBClient) init() {
 	// hammering from a cluster of identical LBClients.
 	cc.nextIdx = uint32(time.Now().UnixNano())
 }
+// clients update: add & remove
+func (cc *LBClient) AddClients(c *HostClient) int {
+    cc.cs = append(cc.cs, &lbClient{
+        c:           c,
+        healthCheck: cc.HealthCheck,
+    })
+    return len(cc.cs)
+}
+func (cc *LBClient) RemoveClients(index int) int {                                                                                                           
+    cc.cs = append(cc.cs[:index], cc.cs[index+1:]...)
+    return len(cc.cs)
+}
```

Comment:
I don't really like removing by index. When you remove one the indexes change so all other `RemoveClients` won't do what you might think they do.

Would it be possible to remove by `Addr` for example where you loop over the HostClients and remove the one with the matching Addr. Or maybe add a callback to `RemoveClient` like `func(*HostClient) bool` so the user can decide which ones to remove?